headers/functions/overlay29.h

```diff
@@ -432,11 +445,17 @@ void SetShouldBoostHiddenStairsSpawnChance(bool value);
 void UpdateShouldBoostHiddenStairsSpawnChance(void);
 bool IsSecretRoom(void);
 bool IsSecretFloor(void);
+bool HiddenStairsPresent(void);
+void HiddenStairsTrigger(bool show_message);
 undefined4 GetDungeonGenInfoUnk0C(void);
 struct minimap_display_data* GetMinimapData(void);
+void DrawMinimapTile(int x, int y);
+void UpdateMinimap(void);
 void SetMinimapDataE447(uint8_t value);
 uint8_t GetMinimapDataE447(void);
 void SetMinimapDataE448(uint8_t value);
+void InitWeirdMinimapMatrix(void);
+void InitMinimapDisplayTile(struct minimap_display_tile* minimap_display_tile, void* ptr);
```

Comment:
I generally reserve `void*` when we _know_ the pointer is generic. If it's a pointer to an unknown struct, we should use `undefined*` instead.